google/cloud/storage/internal/hmac_key_requests.cc

```diff
@@ -108,6 +108,14 @@ StatusOr<ListHmacKeysResponse> ListHmacKeysResponse::FromHttpResponse(
   return result;
 }
 
+std::ostream& operator<<(std::ostream& os, ListHmacKeysResponse const& r) {
+  os << "ListHmacKeysResponse={next_page_token=" << r.next_page_token
+     << ", items={";
+  std::copy(r.items.begin(), r.items.end(),
+            std::ostream_iterator<HmacKeyMetadata>(os, "\n  "));
```

Comment:
Should there be newlines in a rendering?  At least without some highlighting "NOTE:"?

Reply:
No opinion.  What do you prefer?

Reply:
My first choice would be just comma separators, so we'd get something like ...

`ListHmacKeysResponse={next_page_token=blah, items={HmacKeyMetadata={...}, HmacKeyMetadata={...}}}`

That might get longish, but so be it unless/until we come up with some streaming format policy, like ...

```
ListHmacKeysResponse={
  next_page_token=blah, 
  items={
    HmacKeyMetadata={
      ...
    },
    HmacKeyMetadata={
      ...
    }
  }
}
```

Reply:
Went with the commas.